controllers/vrg_volrep.go

```diff
@@ -2024,9 +2024,24 @@ func (v *VRGInstance) validateExistingPVC(pvc *corev1.PersistentVolumeClaim) err
 		return fmt.Errorf("failed to get existing PVC (%w)", err)
 	}
 
-	if existingPVC.ObjectMeta.Annotations == nil ||
-		existingPVC.ObjectMeta.Annotations[RestoreAnnotation] != RestoredByRamen {
-		return fmt.Errorf("found PVC object not restored by Ramen for PVC %s", existingPVC.Name)
+	// If not Bound return error
+	if existingPVC.Status.Phase != corev1.ClaimBound {
+		return fmt.Errorf("PVC exists and is not bound (phase: %s)", existingPVC.Status.Phase)
+	}
+
+	// If terminating return error
+	if !existingPVC.DeletionTimestamp.IsZero() {
+		return fmt.Errorf("existing bound PVC %s deletion timestamp non-zero", existingPVC.GetName())
+	}
+
+	// Check match to the PV we expect PVC to be bound to
+	if existingPVC.Spec.VolumeName != pvc.Spec.VolumeName {
```

Comment:
Had changed this from == to != and forgot to return an error, fixing the same as well.

Reply:
Addressed